### PR TITLE
New data for individual Indiana counties

### PR DESCRIPTION
**Pull request by migurski**
Added so far:

- [x] Allen: https://gis.acimap.us/projectservicesadmin/rest/services/Accela/Accela_Production/MapServer/51
- [x] Hamilton: http://gis.hamiltoncounty.in.gov/ArcGIS/rest/services/Precincts/MapServer/0
- [x] Elkhart: https://elb2.39dn.com/arcgis/rest/services/eGISDynamicServices/ElkhartINDynamic/MapServer/103
- [x] St. Joseph: http://stjosephcountyindiana.com/788/Layer-List
- [x] Document sources in Indiana README

To do:

- [ ] More from largest-population counties at https://en.wikipedia.org/wiki/List_of_counties_in_Indiana

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -463,22 +463,34 @@ out/17-illinois/state.gpkg: data/17-illinois/statewide/2010/tl_2012_17_vtd10.zip
...
+		data/18-indiana/097-marion/2012/precincts.geojson \
+		data/18-indiana/157-tippecanoe/2016/precincts.geojson
 	mkdir -p out/18-indiana/source
 	unzip -d out/18-indiana/source data/18-indiana/statewide/2010/tl_2012_18_vtd10.zip
-	# Skip Tippecanoe and Marion Counties (FIPS 157, 097) since they'll come from another file.
+	# Skip a few since they'll come from another file.
 	# GPKG are weird
 	rm -f $@
-	ogr2ogr -sql "SELECT '2010' AS year, STATEFP10 AS state, COUNTYFP10 AS county, GEOID10 AS precinct, 'polygon' AS accuracy FROM tl_2012_18_vtd10 WHERE COUNTYFP10 != '157' AND COUNTYFP10 != '097'" \
+	ogr2ogr -sql "SELECT '2010' AS year, STATEFP10 AS state, COUNTYFP10 AS county, GEOID10 AS precinct, 'polygon' AS accuracy FROM tl_2012_18_vtd10 WHERE COUNTYFP10 != '003' AND COUNTYFP10 != '057' AND COUNTYFP10 != '097' AND COUNTYFP10 != '157'" \
```
Comment (nvkelso) [inline Makefile]: Please convert this to a alphabetical NOT IN list.

**Comment by nvkelso** [comment]
Please also add/modify the README.md file in the indiana dir explaining the sources (especially how you might have scraped the Esri endpoints for the GeoJSON).

**Comment by migurski** [comment]
Preview for commit 93dcb0dd2f61c5a7ea31baa02c35721c8e0bbf14:

**Comment by nvkelso** [comment]
About 30 minutes for the preview image to show up, not bad!

**Comment by migurski** [comment]
Yeah, with most of the files already checked-out the actual processing doesn't take too long!

**Comment by migurski** [comment]
Preview for commit 62008572bf9ac92890b7cfae324a9639a03774e5:

**Comment by nvkelso** [comment]
Now just for Arizona, Missouri, Alabama, and New York and everything will
be connected, lol.

**Comment by migurski** [comment]
Ha ha yeah, easy.

I’m looking at more of these counties by population. Lake County (Gary, IN) is a big one seemingly without shapefiles or an ArcGIS server, but [they do have PDF maps available](https://www.lakecountyin.org/portal/media-type/html/group/voters/page/default.psml/js_pane/P-144513c72a8-22548). 🤔

**Comment by nvkelso** [comment]
The following link worked better for me:

-
https://www.lakecountyin.org/portal/media-type/html/group/voters/page/default.psml/js_pane/P-144513c72a8-22548

It does say GIS data, probably worth a quick email to them asking for the
source.

**Comment by migurski** [comment]
I sent them a friendly email to see if they publish SHP or ArcGIS anywhere.

**Comment by nvkelso** [comment]
Gah, GitHub keeps holding individual comments until I've submitted a formal review :\

**Comment by migurski** [comment]
Preview for commit a62ecd68f73f2aa3dfeaaa1806966bd679a89468:

**Comment by migurski** [comment]
I think I’ve addressed your comments. I’d like to keep this PR open a little while, to see if Lake county responds and to add a few more data sources.

**Comment by nvkelso** [comment]
@migurski You still want to keep this PR open? Let's set a deadline of this Friday to merge it. Further work can be performed in a new PR...

**Comment by migurski** [comment]
That works for me! There are a solid set of changes here that I’d be happy to merge.

**Pull request merged**